packages/flab/LAB.src.js

```diff
@@ -149,8 +150,14 @@
 					script.src = src;
 					// NOTE: no append to DOM yet, appending will happen when ready to execute
 				}
-				// same-domain and XHR allowed? use XHR preloading
-				else if (preload_this_script && src.indexOf(root_domain) == 0 && chain_opts[_UseLocalXHR]) {
+
+				// This is the ultimate fallback in React Server.  The
+				// "cache-preloading" option in stock LABjs doesn't work in
+				// modern Chrome, so... this is our last best hope.  If you're
+				// configured for splitJsLoadFromExecution then you'd better
+				// have xhr access to your scripts!  They need to either be on
+				// the same domain or have CORS headers.
+				else if (chain_opts[_UseCORSXHR] || (src.indexOf(root_domain) == 0 && chain_opts[_UseLocalXHR])) {
```

Comment:
I only have a cursory familiarity with LABjs, but I wonder if this comment is discoverable enough, since I'm not sure it's obvious that you need xhr access in order to use this (though I would imagine access would generally be granted unless it's cross-domain or some such). Will this actually pop up for people in the real world?